### ISSUE TITLE
index corruption

### ISSUE THREAD

**Issue by flexiblestoic** [kopia/kopia#1939]
Hi all,

KopiaUI version 0.10.7 windows.
I have had this issue pop up on the maintenance task of a snapshot:

`builder: error getting index "ne7a04270ffde4290e12a214189f792f4-s31c2f4222e6f71ea10f": error decrypting BLOB ne7a04270ffde4290e12a214189f792f4-s31c2f4222e6f71ea10f: unable to decrypt content: cipher: message authentication failed.` 

This error persists after a full maintenance. 
I don't know the significance of this. Is there a way to correct the corruption?

Thanks
Chris

**Comment by jkowalski** [comment]
This looks like a partial file write or on-disk corruption. From the file name it appears to be a quite old repository (without epoch blobs). Deleting the index file might fix the issue. 

Did you get any power failures or crashes etc around the time the file was written?

**Comment by flexiblestoic** [comment]
Hi Jarek, 

Thanks, this worked! 
For reference things I tried before: maintenance, full maintenance, index reconstruction.

> Did you get any power failures or crashes etc around the time the file was written?

That is possible, KopiaUI is running in background and sometimes, I just put the computer to sleep without checking maybe this happened in the middle of a backup.


> quite old repository

Yes, this has been running well since 5/2021. I just migrated the repository.

Thanks Jarek for your help
